controllers/AnswerController.js

```diff
@@ -0,0 +1,71 @@
+var flash = require('../utilities/flash.js').flash;
+var Model = require('../models/models.js');
+
+
+module.exports = {
+
+ registerAnswer:  function(req, res){
+  var id_question=req.query.question_id,
+      people_answer=req.body.option,
+      answer = new Model.AnswerModel({
+       id_question:id_question,
+       answer: people_answer
+      });
+  console.log("people anser "+people_answer)
+
+  answer.save(function(error){
+   if(error){
+    flash(req, 'danger', 'registration error!', 'your vote was not registered! try again.');
```

Comment:
you also need to send an answer to the user if the save action fails... On this case you are setting a flash message but you are not rendering or redirecting... So the user will never receive a response